main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Printf("%s")
+}
+
+func Exported() {
```

Comment:
__[golint]__
* [golint] exported function Exported should have comment or be unexported